ci-operator/step-registry/mirror-images/icsp-olm-operators/mirror-images-icsp-olm-operators-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+mirror_auths="${SHARED_DIR}/mirror_auths"
+
+# private mirror registry host
+# <public_dns>:<port>
+MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
+if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
+    echo "File ${SHARED_DIR}/mirror_registry_url does not exist."
+    exit 1
+fi
+echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
+
+# add the custom registry auth to the .dockerconfigjson of the cluster
+registry_cred=`head -n 1 "/var/run/vault/mirror-registry/registry_creds" | base64 -w 0`
+jq --argjson a "{\"${MIRROR_REGISTRY_HOST}\": {\"auth\": \"$registry_cred\"}}" '.auths |= . + $a' "/var/run/vault/mirror-registry/dockerconfigjson" > "${mirror_auths}"
```

Comment:
since `dockerconfigjson` is only for OLM operators in post action, I think it's better to make the name more clear, e.g. `olm_dockerconfigjson`, wdyt?

Reply:
This file was created by running `oc extract secret/pull-secret -n openshift-config  --confirm`, it contains all basic auths of the cluster. It not only used by OLM.

Reply:
afaik, `dockerconfigjson` is only used in this step, will it be used in another step?

Reply:
Yes, I guess so. If others need the cluster auths, they can resue it. But, if we change the file name to `olm_dockerconfigjson`, they will miss it.